documentation/getdata.md

```diff
@@ -1,48 +1,44 @@
 # Get Data
 
-## List
+In order to help grant-makers, developers and analysts discover data that uses the 360Giving Standard, 360Giving maintains a register of data at [http://data.threesixtgiving.org/](http://data.threesixtgiving.org/)
```

Comment:
So 360Giving always write grantmakers as one word. not grant makers, no grant-makers. Just trying to consistent with this.